components/dial/stories/dial.stories.js

```diff
@@ -52,23 +52,26 @@ export default {
 			},
 			control: "boolean",
 		},
+		reducedMotion: { table: { disable: true } },
```

Comment:
```suggestion
```

This is no longer used because reducedMotion is a global setting now.

Reply:
(applies to other stories too)

Reply:
Ah yes, thank you!